RFCs/HDF5_Library/VFD_Subfiling/user_guide/main.tex

```diff
@@ -35,10 +35,10 @@
 % Document metadata: Title, author(s), etc
 \title{HDF5 Subfiling VFD User's Guide}
 \author{Jordan Henderson}
-\date{Jan. 25, 2023}
+\date{Jun. 19, 2023}
```

Comment:
Can't you use \today for the date?

Reply:
Probably yes. This document is just based off our RFC template, so we could probably update that as well.